fluent-package/apt/systemd-test/update-from-v4.sh

```diff
@@ -4,6 +4,11 @@ set -exu
 
 . $(dirname $0)/../commonvar.sh
 
+if [ "${code_name}" == "bookworm" ]; then
+    echo "As bookworm is not published for v4, so upgrade from v4 check is disabled"
+    exit 0
+fi
+
```

Comment:
How about managing which tests should be run in `test.sh` together instead of each test?

```diff
diff --git a/fluent-package/apt/systemd-test/test.sh b/fluent-package/apt/systemd-test/test.sh
index fd862c6..e45005b 100755
--- a/fluent-package/apt/systemd-test/test.sh
+++ b/fluent-package/apt/systemd-test/test.sh
@@ -19,11 +19,16 @@ fi
 set -eu
 
 test_filenames=(
-    update-from-v4.sh
     update-to-next-version.sh
-    update-to-next-version-with-backward-compat-for-v4.sh
 )
 
+if [ $vm != "debian-bookworm" ]; then
+    test_filenames+=(
+        update-from-v4.sh
+        update-to-next-version-with-backward-compat-for-v4.sh
+    )
+fi
+
 for apt_repo_type in local v5 lts; do
     echo -e "\nRun test: $apt_repo_type\n"
     vagrant up $vm
```

Reply:
It may be reasonable to do so. Thanks.